bin/cli.js

```diff
@@ -118,8 +128,13 @@ async.waterfall([
 
       return documentedVersions;
     });
+    const latest = semver.getGreaterVersion(versions);
+    if (argv['get-current']) {
+      console.log(latest);
+      return callback(STOP);
```

Comment:
Don't understand the use of this `Stop` error class, why not just `callback(null)`?

Reply:
This is to avoid printing`Done` at the end, since we want to use this feature in automation it doesn't make sense to add extra input we have to throw away.

Reply:
We do, but it could also be used as a CLI command still. So I'd either propose removing `Done` altogether or not worrying about it! If an error is percolated up and you're running this as an `exec()` process, for example, you'll still get an error code returned rather than having to worry about `stdout` which you can just ignore!

Reply:
Or perhaps another command-flag to turn off the "Done" output? :man_shrugging: